pithos/application.py

```diff
@@ -85,6 +87,38 @@ def do_command_line(self, command_line):
         # http://stackoverflow.com/questions/1943747/python-logging-before-you-run-logging-basicconfig
         logging.root.handlers = []
 
+        # Show the Pithos log since last reboot and exit
+        if options.contains('last-logs'):
+            try:
+                from systemd import journal
+                from os.path import basename
+            except ImportError:
+                type(command_line).do_print_literal(command_line, 'Systemd Python module not found')
```

Comment:
OK. Also I noticed something. since it's a plugin it doesn't start logging until the plugin loads. I'm working on a hand off. Basically it will log like it does currently and when the logging plugin is loaded initial logging is disabled, and then picked back up by the logging plugin(if it's enabled)